cpp/benchmarks/string/combine_benchmark.cpp

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
Why this file has name "combine_...", not "concatenate...."? It is for concatenate benchmark, isn't it?

Reply:
My intention was to add the `cudf::strings::join` to this as well in a future PR.

Reply:
I see. I'm going to write a "suggestion" for this too 😄